dev-shipperhq/docs/insight/faq.md

```diff
@@ -1,9 +1,8 @@
 ---
-sidebar_position: 4
+sidebar_position: 10
```

Comment:
Looks like some of these numbers are too high. The `sidebar_position` represents the order of the the doc on the sidebar. For this section, you would pick a number between 1-5.  

<img width="298" alt="Screen_Shot_2022-03-14_at_8_21_59_AM" src="https://user-images.githubusercontent.com/25215449/158180875-10349807-5b8c-4933-83c4-3e20d9546392.png">